src/App/SaveLoadEditPage.xaml

```diff
@@ -7,7 +7,7 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
```

Comment:
Any specific reason to remove this ?
Other pages have this as well. If we are removing it we should do it for other pages as well just for consistency.